webrender/src/device/gfx/device.rs

```diff
@@ -423,7 +429,7 @@ impl<B: hal::Backend> Device<B> {
             }
             .expect("create_command_pool_typed failed");
             unsafe { hal_cp.reset() };
-            let cp = CommandPool::new(hal_cp);
+            let cp = CommandPool::new(hal_cp.into_raw());
```

Comment:
We should not call `into_raw()`. I think we should create the command pool few lines above with [create_command_pool](https://github.com/gfx-rs/gfx/blob/7e86e6d4067e971de570f1fc3184cdb70519143b/src/hal/src/device.rs#L196) instead of `create_command_pool_typed`, because that function is still related to the typed CommandBuffer.